taxrate.go

```diff
@@ -8,6 +8,17 @@ package stripe
 
 import "encoding/json"
 
+type TaxRateTaxType string
+
+const (
+	TaxRateTaxTypeGst      TaxRateTaxType = "gst"
+	TaxRateTaxTypeHst      TaxRateTaxType = "hst"
+	TaxRateTaxTypePst      TaxRateTaxType = "pst"
+	TaxRateTaxTypeQst      TaxRateTaxType = "qst"
+	TaxRateTaxTypeSalesTax TaxRateTaxType = "sales_tax"
+	TaxRateTaxTypeVat      TaxRateTaxType = "vat"
```

Comment:
All of those 3-letter words should be capitalized because they are acronyms.

Reply:
ptal @remi-stripe got all of them this time